web-next/app/copilot/github-copilot/page.module.css

```
@@ -0,0 +1,1108 @@
+}
+
+.badgeDot {
+  display: inline-block;
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: var(--accent);
+  animation: pulseDot 2s ease infinite;
+}
+
+@keyframes pulseDot {
```

Stylelint (17.10.0)

[error] Expected keyframe name "pulseDot" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)